SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -282,10 +282,85 @@ void ASpatialDebugger::OnEntityAdded(const Worker_EntityId EntityId)
 		if (Actor->IsA<APlayerController>())
 		{
 			LocalPlayerController = Cast<APlayerController>(Actor);
+
+			if (!ConfigUIWidget.IsValid())
+			{
+				if (ConfigUIClass != nullptr)
+				{
+					ConfigUIWidget = CreateWidget<USpatialDebuggerConfigUI>(LocalPlayerController.Get(), ConfigUIClass);
+					if (!ConfigUIWidget.IsValid())
+					{
+						UE_LOG(LogSpatialDebugger, Error, TEXT("Couldn't create config UI widget."));
```

Comment:
maybe something along these lines?
```suggestion
						UE_LOG(LogSpatialDebugger, Error, TEXT("SpatialDebugger config UI will not load. Couldn't create config UI widget for class: %s"), GetNameSafe(*ConfigUIWidget));
```